From b38c7cf2df519bea9a1b9e98866f93cf345a6981 Mon Sep 17 00:00:00 2001
From: labomb <labomb@rochester.rr.com>
Date: Sat, 25 Feb 2017 16:12:40 -0500
Subject: [PATCH 01/19] incorrect cflags include path

---
 libairspy/libairspy.pc.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libairspy/libairspy.pc.in b/libairspy/libairspy.pc.in
index 5a29a83..95c1e93 100644
--- a/libairspy/libairspy.pc.in
+++ b/libairspy/libairspy.pc.in
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: AirSpy Library
 Description: C Utility Library
 Version: @VERSION@
-Cflags: -I${includedir}/ @AIRSPY_PC_CFLAGS@
+Cflags: -I${includedir}/libairspy/ @AIRSPY_PC_CFLAGS@
 Libs: -L${libdir} -lairspy
 Libs.private: @AIRSPY_PC_LIBS@
-- 
2.20.1

