From 5c17bbc14bb548871bb26ffec9509f8f158077fd Mon Sep 17 00:00:00 2001
From: Youssef <youssef@airspy.com>
Date: Thu, 8 Oct 2015 17:36:16 +0200
Subject: [PATCH 08/10] Fixed a c/p error.

---
 libairspy/src/iqconverter_float.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libairspy/src/iqconverter_float.c b/libairspy/src/iqconverter_float.c
index ee34d44..59c160b 100644
--- a/libairspy/src/iqconverter_float.c
+++ b/libairspy/src/iqconverter_float.c
@@ -306,7 +306,7 @@ static void remove_dc(iqconverter_float_t *cnv, float *samples, int len)
 		*samples++ = sample;
 	}
 
-	avg = cnv->avg;
+	cnv->avg = avg;
 }
 
 static void translate_fs_4(iqconverter_float_t *cnv, float *samples, int len)
-- 
2.1.4

