From 8a46b25a9ae8681b2af1bc2f71602b248d7420bd Mon Sep 17 00:00:00 2001
From: bvernoux <bvernoux@gmail.com>
Date: Tue, 24 Nov 2015 19:36:49 +0100
Subject: [PATCH 10/10] Fix Buffer overflow #30 thanks to Nicolasp38

---
 libairspy/src/airspy.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libairspy/src/airspy.c b/libairspy/src/airspy.c
index f215e05..1f68da4 100644
--- a/libairspy/src/airspy.c
+++ b/libairspy/src/airspy.c
@@ -1260,7 +1260,7 @@ extern "C"
 			if(length > 0)
 			{
 				memcpy(version, version_local, length-1);
-				version[length] = 0;
+				version[length-1] = 0;
 				return AIRSPY_SUCCESS;
 			}
 			else
-- 
2.1.4

