From eee8452caf92ff53077ea1a855e67dba4769ab8d Mon Sep 17 00:00:00 2001
From: Thierry Leconte <23015573+TLeconte@users.noreply.github.com>
Date: Sun, 25 Feb 2018 20:21:26 +0100
Subject: [PATCH 12/12] better paramater bound checking for set gain functions
 (#55)

---
 libairspy/src/airspy.c | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/libairspy/src/airspy.c b/libairspy/src/airspy.c
index 555ba45..5aefdd2 100644
--- a/libairspy/src/airspy.c
+++ b/libairspy/src/airspy.c
@@ -1757,16 +1757,13 @@ int airspy_list_devices(uint64_t *serials, int count)
 	int ADDCALL airspy_set_linearity_gain(struct airspy_device* device, uint8_t value)
 	{
 		int rc;
-		value = GAIN_COUNT - 1 - value;
 
 		if (value >= GAIN_COUNT)
 		{
 			value = GAIN_COUNT - 1;
 		}
-		else if (value < 0)
-		{
-			value = 0;
-		}
+
+		value = GAIN_COUNT - 1 - value;
 
 		rc = airspy_set_mixer_agc(device, 0);
 		if (rc < 0)
@@ -1794,16 +1791,13 @@ int airspy_list_devices(uint64_t *serials, int count)
 	int ADDCALL airspy_set_sensitivity_gain(struct airspy_device* device, uint8_t value)
 	{
 		int rc;
-		value = GAIN_COUNT - 1 - value;
 
 		if (value >= GAIN_COUNT)
 		{
 			value = GAIN_COUNT - 1;
 		}
-		else if (value < 0)
-		{
-			value = 0;
-		}
+
+		value = GAIN_COUNT - 1 - value;
 
 		rc = airspy_set_mixer_agc(device, 0);
 		if (rc < 0)
-- 
2.11.0

